### ISSUE TITLE
Add JSON utilities to navigation

### ISSUE THREAD

**Issue by shaedrich** [foundeo/cfdocs#514]
Would it be useful to add "Helpful links" with "[JSON utilities](https://cfdocs.org/utilities/json/)" item to navigation?

----

Anywhere you mentioned a link where contributors can view their contributions if not shown on the main page of cfdocs.org. I can't find this anymore. Did something changed?

**Comment by pfreitag** [comment]
Hi - it is currently mentioned in the JSON file documentation readme here: https://github.com/foundeo/cfdocs#examples I think that is the appropriate place for it, but if you can think of a way to make it more prominent there that is fine with me.

For contributions views you can see that on Github, here: https://github.com/foundeo/cfdocs/graphs/contributors then for a specific user here: https://github.com/foundeo/cfdocs/commits?author=shaedrich

**Comment by shaedrich** [comment]
I guess, it's right to have it on github but in it's current place it's underneath everything else so contributors have to read everything (they should do so but I admit that I did not know the link since you mentioned it yesterday).

That was not what I meant. Directly on cfdocs.org you display contributors and somewhere you mentioned that, if it's not updated because it's cached, contributors can look for their contributions using that tool (it was not one of the links you mentioned above in your answer)...

**Comment by pfreitag** [comment]
Yes the https://cfdocs.org/ main page lists all the contributors, you are showing up there for me, but it does get cached for 24 hours I think so you will not see changes right away. Also if there was an error in the API request to github to get the data then it would get cached without the contributors listed

**Comment by shaedrich** [comment]
I'll see if the changes are applied tomorrow^^

**Comment by shaedrich** [comment]
The changes are applied. But you offered a link in case they would not somewhere. I just forget where that was and how to explain it^^

**Comment by shaedrich** [comment]
Maybe add this link to leaderboard?
